mock-thunderhead/src/main/resources/application.yml

```diff
@@ -36,6 +36,7 @@ auth:
     environment.experience.deletion.enable: true
     endpointgateway.enable: false
     cm.sync.command.poller.enable: true
+    user.sync.credentials.update.optimization.enable: false
```

Comment:
we usually enable new feature in the mock by default so developers would test it. Is there any reasons against turning it on?

Reply:
Enabled.